SKILL.tsv

```diff
@@ -6236,8 +6236,8 @@ SKILL_20160224_006235
 SKILL_20160224_006236	STR: +#{CaptionRatio}#{nl}AoE Attack Ratio: +#{CaptionRatio2}#{nl}Duration: #{CaptionTime}# seconds{nl}Glyph Duration: 15 seconds
 SKILL_20160224_006237
 SKILL_20160224_006238	Attack: #{SkillAtkAdd}#{nl}Attribute Damage: #{CaptionRatio2}#%{nl}Additional Damage to Plant-type Monsters: +#{CaptionRatio}#{nl}Statue Materials Drop Rate: #{CaptionTime}#%
-SKILL_20160224_006239
-SKILL_20160224_006240	
+SKILL_20160224_006239	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}Strongly smacks down the enemy. Deals double damage to mutant and devil-type enemies.
```

Comment:
That skill was already half-translated so I just copied the first half. I thought it was alright.

Maybe this would be better if `Strongly` doesn't sound good to you:

> Smack down the enemy with a powerful attack.

or maybe even `Strike down`.